types/next-auth.d.ts

```diff
@@ -11,4 +12,6 @@ declare module "next-auth" {
 			auth_info?: AuthInfo
 		}
 	}
+
+	type AuthProviderType = BuiltInProviderType | 'bitbucket'
```

Comment:
so, bitbucket is not listed in the `BuiltInProviderType`?

Reply:
Yes, it is not listed. Next-auth does not provide login with Bitbucket out of the box. We had to implement it.

Reply:
Okayy